packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -60,16 +68,11 @@ export function menuItemTemplate(
         ${when(
             x => x.hasSubmenu,
             html<FASTMenuItem>`
-                <div
-                    part="expand-collapse-glyph-container"
-                    class="expand-collapse-glyph-container"
-                >
-                    <span part="expand-collapse" class="expand-collapse">
-                        <slot name="expand-collapse-indicator">
-                            ${options.expandCollapseGlyph || ""}
-                        </slot>
-                    </span>
-                </div>
+                <span part="submenu-icon" class="submenu-icon">
+                    <slot name="submenu-icon">
+                        ${options.submenuIcon || submenuIcon}
+                    </slot>
+                </span>
```

Comment:
@EisenbergEffect do you think this and the below code snippet might be a good candidate for one of the new conditional directives?

Reply:
Note that the expanded check changes with the menu updates i have in the RFC for removing anchored region...

Reply:
It's actually simpler than that I think. We don't need render to do this. Instead, we can have a single computed property that based on the conditions returns just one template. That will be cheaper than multiple `when` instances.